_schemaV2.graphql

```diff
@@ -10075,6 +10075,7 @@ type MarketingCollection {
   showHeaderArtworksRail: Boolean!
   slug: String!
   thumbnail: String
+  thumbnailImage: Image
```

Comment:
**suggestion**: Awesome that we can upgrade the image URL here. What do you think of marking `thumbnail` as deprecated in favor of `thumbnailImage.url` or `thumbbailImage.imageURL`? Is there anyway to do that in Metaphysics or would that have to be a Gravity GraphQL schema change?

Reply:
good idea to deprecate old field. I couldn't find a way how to deprecate a field in the stitching code. I think we can do some manual transformations, but I would better submit a PR to gravity. How does it sound?

Reply:
Sounds good 👍🏼